src/test/java/LoginTests.java

```diff
@@ -17,7 +17,7 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver(options);
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://test.testpro.io/";
+        String url = "https://testpro.io/";
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`

![image](https://user-images.githubusercontent.com/104761482/209428163-6311d74d-be0e-481c-8572-f70201317e43.png)

Reply:
I'm confused, what do you mean methods? We were spouse to use some method to do this task? 
also your example is the same as in the picture thats written I'm not understanding what i did wrong.

Reply:
Don't worry! Everything is fine! I showed an example of how to name a method according to the Java naming convention https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
oh okay, so did i do the hw right way or wrong way?